gemet/thesaurus/models.py

```diff
@@ -363,6 +466,62 @@ class AsyncTask(models.Model):
     )
 
 
+class TimeTrackedModel(models.Model):
+    class Meta:
+        abstract = True
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+
+class Import(TimeTrackedModel):
+    """
+    Keeps track of data imports via excel files.
+    """
+    spreadsheet = models.FileField(
+        upload_to='imports/',
+        help_text=mark_safe(
+            'Details about the supported file format can be found <a href="'
+            'https://taskman.eionet.europa.eu/projects/infrastructure/wiki/'
+            'Importing_new_concepts">here</a>.'
```

Comment:
Change to `Importing_new_concepts_in_GEMET`, I have renamed the page and set the proper parent page